src/components/CreateArticle.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Editor } from 'react-draft-wysiwyg';
+import {
+  Form,
+  Button,
+  Col,
+  Row,
+  Container,
+  OverlayTrigger,
+  Popover,
+} from 'react-bootstrap';
+import { WithContext as ReactTags } from 'react-tag-input';
+import debounce from 'lodash.debounce';
+import uploadImageCallBack from '../utils/imageUpload';
+
+const CreateArticle = ({
+  onSubmit,
+  onEditorStateChange,
+  handleChange,
+  tags,
+  draftStatus,
+  handleDelete,
+  handleAddition,
+  handleDrag,
+  onPublish,
+  show,
```

Comment:
@RachelleMaina, it is better to parse props and destructure instead, it makes it more maintainable.